src/main/java/frc/robot/controllers/SparkMaxPositionController.java

```diff
@@ -212,6 +212,9 @@ public void update() {
       		 case HOMED:
       			 updateRequestedPosition();
   	    }
```

Comment:
Personally, I like to see a break statement inside the last case; but since it has not impact, I'll approve.